hidden.js

```diff
@@ -9,26 +9,20 @@
   }, prefs => {
     chrome.tabs.onUpdated.removeListener(hidden.observe);
     if (prefs['go-hidden']) {
-      chrome.tabs.onUpdated.addListener(hidden.observe);
+      chrome.tabs.onUpdated.addListener(hidden.observe, { properties: ["hidden"] });
```

Comment:
Chrome does not support this, right?

![screen shot](https://user-images.githubusercontent.com/14104614/50375220-f825e100-05f1-11e9-82ad-e4a78c1be440.png)

Reply:
No it does not.
But this line should only be reached on Firefox after these changes.

AFAIK Chrome does not support hidden tabs at all so the code only has to work on Firefox.
And since Firefox supports restricting events to only those you are interested in, that can be used here.